mautic/mautic
# Added merge contacts to contact edit API action

| Q  | A
| --- | ---
| Bug fix? | y
| New feature? | n
| Related user documentation PR URL | n
| Related developer documentation PR URL | n
| Issues addressed (#s or URLs) | #3108
| BC breaks? | n
| Deprecations? | n

[//]: # ( Required: )
#### Description:
Added merge contacts to contact edit API.

#### Steps to test this PR:
1. Create contact with email address
2. Go to website with tracking code and clear cookies on tracking url and mautic url.
3. Add to website with tracking code edit  action https://developer.mautic.org/#edit-contact

```
if(isset($_COOKIE['mtc_id'])){
$contactApi->edit($_COOKIE['mtc_id'], $data);
}
```
4. See if older contact was marger with current mtc_id on the tracking site.
5. _Edit by @escopecz: Run the [API Library test](https://github.com/mautic/api-library/pull/105)_

~~Doesn't work for create action, logical.~~ _Edit by @escopecz: works for edit and create_
